src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] The App.js component should be class-based, kindly consider implementing this so the component is class-based